src/commands/listflows.js

```diff
@@ -1,43 +1,76 @@
 module.exports = {
   description: "Get a list of all the flows.",
-  usage: {},
-  examples: {},
-  aliases: [ "flows" ],
+  usage: {
+    "[flowID]": "ID of a flow"
+  },
+  examples: {
+    "hY5C4i": "Get information of the flow with ID `hY5C4i`"
+  },
+  aliases: [ "listflow", "flows", "flow" ],
   permissionRequired: 1, // 0 All, 1 Mods, 2 Admins, 3 Server Owner, 4 Bot Admin, 5 Bot Owner
-  checkArgs: (args) => !args.length
+  checkArgs: (args) => args.length <= 1
 };
 
 const { limitFlows, formatExplanation, generateTip } = require("../constants/index.js"), config = require("../../config.json");
 
-module.exports.run = async (message, _, gdb, { prefix }) => {
+module.exports.run = async (message, _, gdb, { prefix, content: query }) => {
+  query;
```

Comment:
?

Reply:
I just put that there cuz eslint was bugging me for `query` not being used when I working on something else. Forgot to remove it...